### PR TITLE
created mealsList component and fetch data. Homework week3, meal-sharing  part

### PR DESCRIPTION
**Pull request by AnastasiiaYaitska**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -21,10 +24,54 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
+///
+// router.use("/reservations", reservationsRouter);
+
+router.get("/:type", cors(), async (req, res) => {
```
Comment (LucasIversen) [inline src/backend/app.js:30]: looks good, but we could have also had a path per type, as to not have to do the switch case every time :) Not that it's a very heavy function, but just to keep it as simple as possible

File: src/client/App.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 
 function App() {
   return (
     <Router>
-      <Route exact path="/">
+      <Route exact path="/all-meals">
         <p>test</p>
+        <MealsList />
       </Route>
       <Route exact path="/lol">
```
Comment (LucasIversen) [inline src/client/App.js:13]: Nice route 😄

File: src/client/components/TestComponent/MealsList/MealsList.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  useEffect(() => {
+    const allMeals = async () => {
+      const data = await fetchAllMeals();
+      setMeals(data);
+    };
+    allMeals();
+  }, []);
+
+  if (!meals) {
+    return <p>...Loading</p>;
```
Comment (LucasIversen) [inline src/client/components/TestComponent/MealsList/MealsList.jsx:18]: Good

**Review by LucasIversen — COMMENTED** [review]
Looks good. You've understood the REST api well :) If you want to expand on it, you could try making a post http call for creating meals

File: src/backend/app.js (hunk 1 above)
Comment (AnastasiiaYaitska) [reply src/backend/app.js:30]: Ok. Thank you.I will try fix it in next PR. 😊

**Pull request merged**